server/manage.py

```diff
@@ -19,10 +19,14 @@
 import os
 import sys
 
+from opentelemetry.instrumentation.django import DjangoInstrumentor
 
 def main():
     """Run administrative tasks."""
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "avocano_api.settings")
+
+    DjangoInstrumentor().instrument()
```

Comment:
question: Should this be wrapped in additional controls, for example to not run instrumentation in test?

Reply:
Unlike #291, the server unit tests pass even with this. I'll investigate how this works.